docs/content/configuration/broker.md

```diff
@@ -103,3 +103,4 @@ You can optionally only configure caching to be enabled on the broker by setting
 |`druid.cache.hosts`|Command separated list of Memcached hosts `<host:port>`.|none|
 |`druid.cache.maxObjectSize`|Maximum object size in bytes for a Memcached object.|52428800 (50 MB)|
 |`druid.cache.memcachedPrefix`|Key prefix for all keys in Memcached.|druid|
+|`druid.cache.failureMode`|Failure mode when there is a problem communicating with memcached(`Cancel` or `Retry`)|`Cancel`|
```

Comment:
seems a bit strange that we use upper-case values here, since we always try to use lower-case or camel-case property values. I know it might be a bit of a pain to make things work because of the way enums get de-serialized.